kvmagent/kvmagent/test/unittest_tools/prepare_env.sh

```
@@ -82,12 +83,21 @@ prepare_mn_mock() {
     cd /root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/
+
+    # build pypi source in /root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/pypi
+    mkdir -p ./requirements
+    mkdir -p ./pypi
+    bash pypi_source_builder.sh build \
+        --package-list ./package_list_base.txt --target-path ./pypi --requirement-output-path ./requirements/requirements1.txt
+    bash pypi_source_builder.sh build \
+        --package-list ./package_list_virtualenv.txt --target-path ./pypi --requirement-output-path ./requirements/requirements2.txt
+    cat requirements/requirements1.txt
+    cat requirements/requirements2.txt
     pip install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)